apps/web/src/routes/AuthLayoutRouter/AuthLayoutRouter.js

```diff
@@ -64,7 +64,7 @@ import ThreadList from 'routes/Messages/ThreadList'
 import UserSettings from 'routes/UserSettings'
 import { GROUP_TYPES } from 'store/models/Group'
 import classes from './AuthLayoutRouter.module.scss'
-import { findHomeView } from 'util/contextWidgets'
+import { findHomeView } from '@hylo/shared/src/ContextWidgetPresenter'
```

Comment:
Not clear on what its doing, but this on the surface doesn't look like something I'd not put in the Presenter. Without understanding it better, and assuming it belongs in the ContextWidgetPresenter module as a method outside of the default export of the transformer function then I would name it fully as findHomeWidgetView or something a bit more clearly designating its type amongst the options? What is a view by the way?

Reply:
This is used to determine the default view for a group; so its more of a GroupPresenter thing.

Reply:
yep, that makes sense to me and we'll need this in mobile. so hold that though and once i push down the shared stuff from mobile we'll sort this out.